### PR TITLE
Initial project structure

### PR DESCRIPTION
**Pull request by isimluk**
```
$ kubectl falcon help image-refresh 
Fetches falcon container image from the falcon.crowdstrike.com and pushes it to the container registry or local docker daemon

Usage:
  falcon image-refresh DESTINATION-IMAGE [flags]

Examples:

# Push image directly from CrowdStrike Falcon API to remote registry
REGISTRY_PASSWORD=$(aws ecr get-login-password --region eu-west-1) kubectl falcon image-refresh -u AWS docker://123456.dkr.ecr.REGION.amazonaws.com/falcon-sensor:latest

# Push image from CrowdStrike Falcon API to local docker daemon
kubectl falcon image-refresh docker-daemon:falcon-sensor:latest


Flags:
  -u, --dest-user string   Username for accessing the push registry, use $REGISTRY_PASSWORD env variable to supply password
  -h, --help               help for image-refresh

Global Flags:
      --command-timeout duration   timeout for the command execution
```

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Šimon Lukašík
```
Comment (shawndwells) [inline LICENSE:3]:
```suggestion
Copyright (c) 2021 CrowdStrike
```

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+```
+go get -u -v github.com/crowdstrike/kubectl-falcon
+```
+
+## Usage
+```
+kubectl falcon help
+```
+>>>>>>> 6de642e (Initial project structure)
```
Comment (shawndwells) [inline README.md:27]:
```suggestion
```

**Review by shawndwells — CHANGES_REQUESTED** [review]

**Comment by isimluk** [comment]
Fixed! Thanks!

**Review by shawndwells — APPROVED** [review]

**Comment by redhatrises** [comment]
Nice work! Ack

**Pull request merged**